utils/tables/ui_utils.R

```diff
@@ -329,7 +329,7 @@ tables_controlbar <- function(page, league_config, tables_rv) {
                 column(12,
                        h4("Player Settings"),
                        tables_cb_numeric(page, league_config, tables_rv,
-                                         "Minimum Minutes Played", "minimum_minutes", 25),
+                                         "Minimum Minutes Played", "minimum_minutes", 250),
```

Comment:
What does the `minimum_minutes` here control? What players appear on the web app? Are you changing it for players besides goalkeepers?